presto-main/src/main/java/com/facebook/presto/sql/planner/PlanOptimizers.java

```diff
@@ -167,10 +167,7 @@ public PlanOptimizers(Metadata metadata, SqlParser sqlParser, FeaturesConfig fea
                 new UnaliasSymbolReferences(),
                 new IterativeOptimizer(
                         stats,
-                        ImmutableSet.of(
-                                new RemoveRedundantIdentityProjections(),
-                                new TransformCorrelatedInPredicateToJoin(),
-                                new ImplementFilteredAggregations())
+                        ImmutableSet.of(new RemoveRedundantIdentityProjections())
```

Comment:
@martint , When i was adding correlated IN, i added my rules to existing `RemoveRedundantIdentityProjections`. You can view this as revert+add the rules in a better place.